smugline.py

```diff
@@ -167,7 +180,7 @@ def _get_remote_images(self, album, extras=None):
 
     def _get_md5_hashes_for_album(self, album):
         remote_images = self._get_remote_images(album, 'MD5Sum')
-        md5_sums = [x['MD5Sum'] for x in remote_images['Album']['Images']]
+        md5_sums = [x['MD5Sum'] for x in remote_images['Album']['Images'] if 'MD5Sum' in x]
```

Comment:
Pasting my comment from the PR here -- 

MD5s is that what I was seeing is that API returned elements that were missing the MD5Sum value. When the loop would iterate over it it would crash with a KeyError exception.

There is a chance that you might double upload something, but it's at the expense of crashing.